python/ray/data/dataset.py

```diff
@@ -1952,6 +1985,11 @@ def to_torch(self,
                 if the dataset size is not divisible by the batch size. If
                 False and the size of dataset is not divisible by the batch
                 size, then the last batch will be smaller. Defaults to False.
+            unsqueeze_label_tensor (bool): If set to True, the label tensor
```

Comment:
will name it `squeeze_label_tensor` and default to False sound better?

Reply:
Perhaps, though the operation being performed is unsqueezing, as the label tensors are created squeezed by default (by torch). I feel like `unsqueeze_label_tensor` better conveys the fact that an additional operation is carried out